src/content/CloudBoilerplateNet/Services/DependentTypesResolver.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using CloudBoilerplateNet.Helpers;
+
+namespace CloudBoilerplateNet.Services
+{
+    public class DependentFormatResolver : IDependentTypesResolver
```

Comment:
I'd refactor this to use a dictionary of enumerables

Reply:
Changed to a dictionary. However, the result seems rather error-prone and against DRY.

Reply:
it looks much better now